docs/flashbots-auction/advanced/rpc-endpoint.mdx

```diff
@@ -838,6 +838,37 @@ where
 - `consideredByBuildersAt`: indicates time at which each builder selected the bundle to be included in the target block
 - `sealedByBuildersAt`: indicates time at which each builder sealed a block containing the bundle
 
+### flashbots_setFeeRefundRecipient
+
+The `flashbots_setFeeRefundRecipient` JSON-RPC method allows a user to "delegate" their fee refunds to a specific wallet address.  Two addresses must be provided, the first is the address associated with the signing key used to authenticate your request, while the second is the address to send refunds to.
```

Comment:
Nit: Could be useful to make "fee refunds" a link back to the docs on gas fee refunds here https://docs.flashbots.net/flashbots-auction/advanced/gas-fee-refunds

Reply:
Good call, updated in https://github.com/flashbots/flashbots-docs/pull/567/commits/fccfde4c4da34c86647fb5b3796a64dbd724a687 and tested on vercel.